src/transaction/log_lsa_utils.cpp

```diff
@@ -38,4 +38,10 @@ namespace cublog::lsa_utils
     deserializer.unpack_bigint (big_int);
     lsa = big_int;
   }
+
+  std::size_t
+  get_packed_size (cubpacking::packer &serializator, std::size_t start_offset, std::size_t size)
+  {
+    return serializator.get_packed_bigint_size (start_offset + size);
+  }
```

Comment:
The signature for get_packed_size() functions is serializator & start_offset. The start_offset is the offset to the packed object. Remove the size argument. Caller has to do the sum between another starting point and the offset of the packed log_lsa.
```suggestion
  std::size_t
  get_packed_size (cubpacking::packer &serializator, std::size_t start_offset)
  {
    return serializator.get_packed_bigint_size (start_offset);
  }
```